src/lib/item-input/object.ts

```diff
@@ -43,7 +43,9 @@ export type ObjectDefOptions<T> = {
 	validateFinal?: (item: T, context?: unknown[]) => true | string
 }
 
-const defaultSummarizeForEditFn = (name: string) => (): string => { throw Error(`missing implementation of summarizeForEdit for objectDef ${name}`) }
+const defaultSummarizeForEditFn = (name: string) => (): string => {
+	throw Error(`missing implementation of summarizeForEdit for objectDef ${name}`)
```

Comment:
Just to double check: this does not want to be a `fatalError`?

Reply:
Correct. I intentionally left this one as an exception because, if it were to happen, it would indicates a bug in the code rather than anything the user should see.